Create condition-operator.md

gomesalexandre opened the pull request:
Same content as the TIL file I created earlier, but would fit here also

gomesalexandre committed:
Create condition-operator.md

Same content as the TIL file I created earlier, but would fit here also !

msachi commented:
I'm not sure this is fit for a README, to be fair, if we are to follow the [guidelines](https://github.com/codingforeveryone/READMEs/blob/master/README.md). But maybe you can find an existing README that this could be added to.

gomesalexandre commented:
Well there is one for the modulo operator, so I guess small topics like this can be added as a README. But I will add some references to it, that's for sure

msachi commented:
Yeah, I was thinking in terms of the sections that every README should have (Introduction, References, Related). Perhaps also a few more examples of use, some katas... I just think READMEs just be more comprehensive than a quick tip.

gomesalexandre committed:
Added related, headers

-Added related section: Kata, and the MDN page of the operator 
-Added headers when needed

gomesalexandre committed:
Update condition-operator.md

Added a second example

gomesalexandre committed:
Update condition-operator.md

Fixed typo
